user-org-actor/src/main/java/org/sunbird/organization/OrgSearchActor.java

```diff
@@ -0,0 +1,43 @@
+package org.sunbird.organization;
+
+import org.sunbird.BaseActor;
+import org.sunbird.OperationValidator;
+import org.sunbird.actor.core.ActorConfig;
+import org.sunbird.actorOperation.OrgActorOperations;
+import org.sunbird.exception.BaseException;
+import org.sunbird.organization.service.IOrgService;
+import org.sunbird.organization.service.OrgServiceImpl;
+import org.sunbird.request.Request;
+import org.sunbird.response.Response;
+
+@ActorConfig(
+  tasks = {"searchOrg"},
+  asyncTasks = {}
+)
+public class OrgSearchActor extends BaseActor {
+
+  private static IOrgService INSTANCE = new OrgServiceImpl();
```

Comment:
manke INSTANCE to small case , As all upper case letters are primarily used to identify constants.